recipes/tempo2/build.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash
+
+./bootstrap
+#export CXXFLAGS=$(echo "$CXXFLAGS" | sed 's/-O2//' | perl -pe 's/-std=.+ /-std=c++98 /')
+#echo "CXXFLAGS $CXXFLAGS"
+
+export TEMPO2=$PREFIX/share/tempo2
+./configure --prefix=$PREFIX --disable-local --disable-psrhome PGPLOT_DIR=$PREFIX/include/pgplot
+make -j2
+make install
+make -j2 plugins
+make plugins-install
+
+# Copy runtime stuff
+for dir in atmosphere ephemeris example_data observatory plugin_data solarWindModel clock earth
+do
+    cp -a T2runtime/$dir $TEMPO2/
+done
+
+# This foo will make conda automatically define a TEMPO2 env variable
+# when the environment is activated.
+etcdir=$PREFIX/etc/conda
+mkdir -p $etcdir/activate.d
+echo "setenv TEMPO2 $TEMPO2" > $etcdir/activate.d/tempo2-env.csh
```

Comment:
Ideally we should create a script that checks if the variable exists, stores its value, sets the conda one and in the deactivation, unset the conda's and re-set the original. With that said, how common is Tempo2 installation from other sources? If it is not that common we can leave as-is.

Reply:
I'll add this in.

Reply:
I've done this now (hopefully in a reasonable way!)

Reply:
See https://github.com/conda-forge/staged-recipes/pull/13221#commitcomment-44879327. But Let's merge this and, if you want to refine it later, you can try it on the feedstock.